### ISSUE TITLE
Elixir does not complain about this atom with quotes, but the return value is without quotes

### ISSUE THREAD

**Issue by eksperimental** [elixir-lang/elixir#8660]
It happens with the  'GREEK CAPITAL LETTER OMEGA' (U+03A9)

```iex
iex(18)> :"Ω"
:Ω
```
It's a nitpick, but it could lead to other bugs.

Just for the record,  there is an equivalent to the omega sign, the OHM SIGN (U+2126), that needs quotes, and it is returned with quotes. In case it has to do with the bug.

```iex
iex(1)> :Ω
** (SyntaxError) iex:1: Elixir expects unquoted Unicode atoms, variables, and calls to be in NFC form.

Got:

    "Ω" (codepoints 2126)

Expected:

    "Ω" (codepoints 03A9)

Syntax error before: Ω

iex(1)> :"Ω"
:"Ω"
```

**Comment by josevalim** [comment]
Yes, we agreed to only warn for ascii because developers complained that it
was hard to track if they needed the quotes or not  in all other cases.
Plus, the reason for the warning is to avoid developers thinking adding
quotes would make them strings.

The case you say can be a bug can happen at any moment, when it comes to
NFC and NFD, and it also happens if you always quote, so it is not related
to quoting per se.
-- 


*José Valim*
www.plataformatec.com.br
Skype: jv.ptec
Founder and Director of R&D

**Comment by josevalim** [comment]
To be more precise, if you always quote, then you still need to know if you
are on NFC or NFD. The nice thing about unquoting is that it tells you in
which format it is. But there is no escape from having to know which is
which.
-- 


*José Valim*
www.plataformatec.com.br
Skype: jv.ptec
Founder and Director of R&D
